release-notes/5.0/preview/api-diff/preview4/Asp.Net/5.0-preview4_Microsoft.Extensions.Caching.Memory.md

```diff
@@ -0,0 +1,31 @@
+# Microsoft.Extensions.Caching.Memory
+
+``` diff
+ namespace Microsoft.Extensions.Caching.Memory {
+     public interface ICacheEntry : IDisposable {
+-        DateTimeOffset? AbsoluteExpiration { get; set; }
++        Nullable<DateTimeOffset> AbsoluteExpiration { get; set; }
```

Comment:
Should these be considered equivalent?

Reply:
They of course are :)  @Anipik, why is the tool generating `Nullable<ValueType>` now instead of `ValueType?`?

Reply:
Even if there was a diff, we should use the `?` syntax to avoid confusion.